packages/tsconfig.base.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowUnreachableCode": false,
+    "allowUnusedLabels": false,
+    "declaration": true,
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["es2016"],
```

Comment:
i don't think we need to bundle the es6 since we target es6,everything should be available at runtime

Reply:
I agree. In general, I think a few rules will change once we work on the TypeScript guidelines if we decide to go with bundling basically nothing and relying on manually provided shims. (manually in the sense that we require them based on conditions, not that the user has to do it)

Reply:
Done.